deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -5000,14 +5000,17 @@ NAN_METHOD(WebGLRenderingContext::GetSupportedExtensions) {
   for (GLint i = 0; i < numExtensions; i++) {
     char *extension = (char *)glGetStringi(GL_EXTENSIONS, i);
     if(extension == webglExtensions[i]){
-      result->Set(i, JS_STR(webglExtensions[i]));
+      int numResults = result->Length();
```

Comment:
it may be redundant to add this assign this to a variable, instead just use `result->Length()+1` instead of `numResults`

Reply:
No worries, this would get optimized away.